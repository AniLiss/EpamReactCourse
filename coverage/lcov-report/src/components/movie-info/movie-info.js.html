<!doctype html>
<html lang="en">
<head>
    <title>Code coverage report for src/components/movie-info/movie-info.js</title>
    <meta charset="utf-8" />
    <link rel="stylesheet" href="../../../prettify.css" />
    <link rel="stylesheet" href="../../../base.css" />
    <meta name="viewport" content="width=device-width, initial-scale=1">
    <style type='text/css'>
        .coverage-summary .sorter {
            background-image: url(../../../sort-arrow-sprite.png);
        }
    </style>
</head>
<body>
<div class='wrapper'>
  <div class='pad1'>
    <h1>
      <a href="../../../index.html">All files</a> / <a href="index.html">src/components/movie-info</a> movie-info.js
    </h1>
    <div class='clearfix'>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Statements</span>
        <span class='fraction'>2/2</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Branches</span>
        <span class='fraction'>0/0</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Functions</span>
        <span class='fraction'>1/1</span>
      </div>
      <div class='fl pad1y space-right2'>
        <span class="strong">100% </span>
        <span class="quiet">Lines</span>
        <span class='fraction'>2/2</span>
      </div>
    </div>
    <p class="quiet">
      Press <em>n</em> or <em>j</em> to go to the next uncovered block, <em>b</em>, <em>p</em> or <em>k</em> for the previous block.
    </p>
  </div>
  <div class='status-line high'></div>
<pre><table class="coverage">
<tr><td class="line-count quiet"><a name='L1'></a><a href='#L1'>1</a>
<a name='L2'></a><a href='#L2'>2</a>
<a name='L3'></a><a href='#L3'>3</a>
<a name='L4'></a><a href='#L4'>4</a>
<a name='L5'></a><a href='#L5'>5</a>
<a name='L6'></a><a href='#L6'>6</a>
<a name='L7'></a><a href='#L7'>7</a>
<a name='L8'></a><a href='#L8'>8</a>
<a name='L9'></a><a href='#L9'>9</a>
<a name='L10'></a><a href='#L10'>10</a>
<a name='L11'></a><a href='#L11'>11</a>
<a name='L12'></a><a href='#L12'>12</a>
<a name='L13'></a><a href='#L13'>13</a>
<a name='L14'></a><a href='#L14'>14</a>
<a name='L15'></a><a href='#L15'>15</a>
<a name='L16'></a><a href='#L16'>16</a>
<a name='L17'></a><a href='#L17'>17</a>
<a name='L18'></a><a href='#L18'>18</a>
<a name='L19'></a><a href='#L19'>19</a>
<a name='L20'></a><a href='#L20'>20</a></td><td class="line-coverage quiet"><span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-yes">2x</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span>
<span class="cline-any cline-neutral">&nbsp;</span></td><td class="text"><pre class="prettyprint lang-js">import React from 'react';
import './movie-info.scss';
&nbsp;
export const Movie = () =&gt; {
    return (
        &lt;section className="movie"&gt;
            &lt;div className="movie__img"&gt;
                &lt;img width="200" height="400" alt="Pulp Fiction" /&gt;
            &lt;/div&gt;
            &lt;div className="movie__info"&gt;
                &lt;h2 className="movie__title"&gt;Pulp Fiction&lt;/h2&gt;
                &lt;h3 className="movie__short-desc"&gt;Oscar Winning Movies&lt;/h3&gt;
                &lt;span className="movie__year"&gt;1994&lt;/span&gt;
                &lt;span className="movie__duration"&gt;154 min&lt;/span&gt;
                &lt;p className="movie__long-desc"&gt;Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.&lt;/p&gt;
            &lt;/div&gt;
        &lt;/section&gt;
    )
}
&nbsp;</pre></td></tr>
</table></pre>
<div class='push'></div><!-- for sticky footer -->
</div><!-- /wrapper -->
<div class='footer quiet pad2 space-top1 center small'>
  Code coverage
  generated by <a href="https://istanbul.js.org/" target="_blank">istanbul</a> at Sun Apr 14 2019 19:16:22 GMT+0300 (Moscow Standard Time)
</div>
</div>
<script src="../../../prettify.js"></script>
<script>
window.onload = function () {
        if (typeof prettyPrint === 'function') {
            prettyPrint();
        }
};
</script>
<script src="../../../sorter.js"></script>
<script src="../../../block-navigation.js"></script>
</body>
</html>
