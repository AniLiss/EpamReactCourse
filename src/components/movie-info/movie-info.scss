.movie {
  display: flex;

  &__year {
    margin-right: 20px;
  }

  &__img {

    img {
      background-color: #ccc;
      display: block;
    }
  }
}
