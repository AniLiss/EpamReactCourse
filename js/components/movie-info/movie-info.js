import React from 'react';
import './movie-info.scss';

export const Movie = () => {
    return (
        <section className="movie">
            <div className="movie__img">
                <img width="200" height="400" alt="Pulp Fiction" />
            </div>
            <div className="movie__info">
                <h2 className="movie__title">Pulp Fiction</h2>
                <h3 className="movie__short-desc">Oscar Winning Movies</h3>
                <span className="movie__year">1994</span>
                <span className="movie__duration">154 min</span>
                <p className="movie__long-desc">Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.</p>
            </div>
        </section>
    )
}
