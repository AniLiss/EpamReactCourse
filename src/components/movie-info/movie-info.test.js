import React from 'react';
import {shallow} from 'enzyme';
import { Movie } from './movie-info';

const wrapper = shallow(<Movie />);

describe('Movie', () => {
    it('is expected to render Movie correctly', () => {
        expect(wrapper).toMatchSnapshot();
    })
});
