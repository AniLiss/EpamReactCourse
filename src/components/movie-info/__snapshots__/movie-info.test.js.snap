// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Movie is expected to render Movie correctly 1`] = `
<section
  className="movie"
>
  <div
    className="movie__img"
  >
    <img
      alt="Pulp Fiction"
      height="400"
      width="200"
    />
  </div>
  <div
    className="movie__info"
  >
    <h2
      className="movie__title"
    >
      Pulp Fiction
    </h2>
    <h3
      className="movie__short-desc"
    >
      Oscar Winning Movies
    </h3>
    <span
      className="movie__year"
    >
      1994
    </span>
    <span
      className="movie__duration"
    >
      154 min
    </span>
    <p
      className="movie__long-desc"
    >
      Sed ut perspiciatis unde omnis iste natus error sit voluptatem accusantium doloremque laudantium, totam rem aperiam, eaque ipsa quae ab illo inventore veritatis et quasi architecto beatae vitae dicta sunt explicabo.
    </p>
  </div>
</section>
`;
